### DISCUSSION TITLE
This repository is over its data quota.

### DISCUSSION THREAD

**Discussion by gitrimius** [shanteacontrols/OpenDeck#69] [Q&A]
Hi,

I was wondering whether there is another way of cloning the repo, since the account is over its quota.
or should I be patient and wait for a while?
tnx a lot!

**Comment by paradajz** [comment]
Hi,

I messed that up while I was setting up new pipelines... You should be able to clone it without issues, only the LFS quota is limited (git lfs pull) which means you don't get touchscreen layouts, some binaries etc. Code should pull without issues (I guess).

**Comment by gitrimius** [comment]
ok.
I get the following messages:

osboxes@osboxes:~$ git clone --recursive https://github.com/paradajz/OpenDeck.git
Cloning into 'OpenDeck'...
remote: Enumerating objects: 31549, done.
remote: Counting objects: 100% (390/390), done.
remote: Compressing objects: 100% (233/233), done.
remote: Total 31549 (delta 183), reused 288 (delta 143), pack-reused 31159
Receiving objects: 100% (31549/31549), 83.15 MiB | 3.94 MiB/s, done.
Resolving deltas: 100% (19476/19476), done.
Downloading bin/touchscreen/design/corel/bergamot.cdr (1.4 MB)
Error downloading object: bin/touchscreen/design/corel/bergamot.cdr (a89592d): Smudge error: Error downloading bin/touchscreen/design/corel/bergamot.cdr (a89592d2f8127ca7d00fa02065f6691c3589c0790d33b1a35dba2983876a9856): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to /home/osboxes/OpenDeck/.git/lfs/logs/20210505T101737.722094868.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: bin/touchscreen/design/corel/bergamot.cdr: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

----------
and

osboxes@osboxes:~/OpenDeck/src$ make TARGET=blackpill411_axo_final
Building application...
Generating board definitions...
Building: board/gen/blackpill411_axo_final/Pins.cpp
In file included from board/gen/blackpill411_axo_final/Pins.cpp:1:
board/gen/blackpill411_axo_final/Pins.h:1:10: fatal error: core/src/general/IO.h: No such file or directory
    1 | #include "core/src/general/IO.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:264: build/app/blackpill411_axo_final/release/board/gen/blackpill411_axo_final/Pins.cpp.o] Error 1
make: *** [Makefile:177: merged] Error 2

**Reply by paradajz** [reply]
Ah well, crap. Clone it like this then:

`GIT_LFS_SKIP_SMUDGE=1 git clone --recursive https://github.com/paradajz/OpenDeck.git`

**Comment by gitrimius** [comment]
Great, worked! tnx!
